migrating_from_ocp_3_to_4/planning-migration-3-4.adoc

```diff
@@ -73,6 +73,82 @@ For more information, see xref:../updating/updating-cluster-between-minor.adoc#u
 
 Review the changes and other considerations that might affect your transition from {product-title} 3.11 to {product-title} 4.
 
+[id="migration-DNS-considerations"]
+=== DNS considerations
+In a typical migration scenario, the DNS domain of the {product-title} 4 target cluster is usually different from the {product-title} 3 source cluster. This implies that the migrated applications have new URLs.
+
+If the source cluster uses the base domain ocp3.example.com and if the default domain of the applications is `\*.apps.ocp3.example.com`, unless special measures are taken, the target cluster is deployed on a different domain,  `ocp4.example.com`, to avoid conflicts with the default applications defined as `\*.apps.ocp4.example.com`.
```

Comment:
`\*.apps.ocp3.example.com`
Why are you escaping an asterisk? What is the purpose?